src/crangler/c_wrangler.cpp

```diff
@@ -41,27 +41,30 @@ struct c_wranglert
   std::vector<std::string> defines;
 
   // transformations
-  struct contract_clauset
+  struct function_contract_clauset
   {
     std::string clause;
     std::string content;
-    contract_clauset(std::string _clause, std::string _content)
+    function_contract_clauset(std::string _clause, std::string _content)
```

Comment:
Not for this PR, but just asking a favour nonetheless: would you be able to dedicate some time to adding function contract tests to Crangler's test suite?

Reply:
Yes, I will create another pull request to add some tests for function contracts.